src/forge/yaml_loader.py

```diff
@@ -141,11 +141,13 @@ def _get_type(x):
         Optional('gpu_flag'): And(bool),
         Optional('market'): And(Or(str, list)),
         Optional('name'): And(str, len, error='Invalid Name'),
+        Optional('on_demand_failover'): And(bool),
```

Comment:
Do you mind adding these parameters in the docs?

Reply:
Just did, sorry for the delay.